src/hkdf.rs

```diff
@@ -15,21 +15,23 @@ use generic_array::{ArrayLength, GenericArray};
 use hmac::{Hmac, Mac};
 
 pub struct Hkdf<D>
-    where D: Digest + Default,
+    where D: Digest,
           D::OutputSize: ArrayLength<u8>
 {
     pub prk: GenericArray<u8, D::OutputSize>,
 }
 
 impl<D> Hkdf<D>
-    where D: Digest + Default,
+    where D: Digest,
```

Comment:
👍

Reply:
to be honest, I just removed that to see if it'd work, I'm not 100% positive it's unnecessary now